app/views/content_items/guide.html+print.erb

```diff
@@ -3,22 +3,52 @@
   content_for :simple_header, true
   content_for :extra_head_content do %>
   <meta name="robots" content="noindex, nofollow">
-  <script>window.onload = function() { window.print(); }</script>
 <% end %>
 
 <div class="govuk-grid-row" id="guide-print">
   <div class="govuk-grid-column-two-thirds">
-    <%= render 'govuk_publishing_components/components/title', { title: @content_item.title } %>
+    <%= render 'govuk_publishing_components/components/title', { 
+      margin_bottom: 6,
+      title: @content_item.title,
+    } %>
+
+    <%= render 'govuk_publishing_components/components/lead_paragraph', {
```

Comment:
Spoke to @jon-kirwan and really like that idea @maxgds. :)

Reply:
Thanks @maxgds - have updated both page / schema types as above. I can't see a way to pass in an extra class to the lead paragraph component either so instead have wrapped it with a `div` and added the `govuk-!-display-none-print` class.

I've updated in the same commit since it's quite a small change.

Also updated the link examples in the description.